src/hooks/useActiveAccount.tsx

```diff
@@ -171,7 +183,7 @@ export const ActiveAccountContextProvider = ({
         accountsWithProduct,
         refetch,
         setActiveAccountId,
-        isLoading: accountsResult.isLoading,
+        isLoading: accountsResult.isLoading || !isSettled,
```

Comment:
Is `isSettled` needed?  `useActiveAccount` should initialize with `isLoading=true`, and RootStack is already paying attention to `isFetched`.

Reply:
Yes it is needed. The purpose of `isSettled` is to sync up the selected account (if there is one) and `isLoading`. 

Before this change isFetched was being set to true and isLoading was set to false before the `useEffect` fired to determine which account to select. In that state, the RootStack logic briefly thought that the accounts had been loaded and that the user did not belong to any accounts and displayed the warning. 

After this change, the RootStack will never get to a state where it thinks the accounts have been loaded without the useActiveAccount logic running and selecting the active account if one exists.